### PR TITLE
Increase in source documentation coverage

### PR DESCRIPTION
**Pull request by SD10**
This should bring coverage of [moya.github.io](https://moya.github.io) up to 93%.

I didn't add all the parameters for all the function declarations. I'm still indecisive. It felt a little bit like boilerplate, so I left them out for now.

Feel free to critique 😃

### CODE REVIEW

File: Sources/Moya/MultiTarget.swift [hunk 1]
```diff
@@ -5,34 +5,43 @@ public enum MultiTarget: TargetType {
...
     public var method: Moya.Method {
         return target.method
     }
 
+    /// The sampleData of the embedded target.
     public var sampleData: Data {
         return target.sampleData
     }
 
+    /// The `Task` of the embedded target
```
Comment (lucas34) [inline Sources/Moya/MultiTarget.swift]: I think this sentence is missing the '.' at the end

File: Sources/Moya/Moya+Alamofire.swift [hunk 2]
```diff
@@ -32,10 +32,12 @@ extension Request: RequestType { }
 public final class CancellableToken: Cancellable, CustomDebugStringConvertible {
     let cancelAction: () -> Void
     let request: Request?
+
     public fileprivate(set) var isCancelled = false
 
     fileprivate var lock: DispatchSemaphore = DispatchSemaphore(value: 1)
 
+
```
Comment (sunshinejr) [inline Sources/Moya/Moya+Alamofire.swift]: Additional new line, I believe not necessary.

File: Sources/Moya/MoyaProvider+Internal.swift [hunk 3]
```diff
@@ -4,6 +4,9 @@ import Result
 // MARK: - Method
 
 extension Method {
+    /// A Boolean value determining whether the request supports multipart.
+    /// True: .post, .put, .patch, .connect
```
Comment (sunshinejr) [inline Sources/Moya/MoyaProvider+Internal.swift]: I would avoid giving implementation details in the comment - once you change the method you would have to remember to change the comment too. First line should be sufficient.

File: Sources/Moya/MoyaProvider.swift [hunk 4]
```diff
@@ -149,19 +168,23 @@ public extension MoyaProvider {
     // Swift won't let us put the StubBehavior enum inside the provider class, so we'll
     // at least add some class functions to allow easy access to common stubbing closures.
 
+    /// A class method that returns StubBehavior.never.
```
Comment (sunshinejr) [inline Sources/Moya/MoyaProvider.swift]: Again, I would avoid implementation details and say what does this stub option mean (in that case it wouldn’t stub at all)

**Comment by SD10** [comment]
Lame.. I forgot this was targeting `master` and not `development`. Have to strip all these commits now 😐

**Comment by SD10** [comment]
Alright, I've removed the comments regarding implementation details, removed the superfluous initializer comments, and fixed the lint warnings.

**Pull request merged**